common/src/main/java/com/alibaba/fescar/common/util/CollectionUtils.java

```diff
@@ -0,0 +1,28 @@
+package com.alibaba.fescar.common.util;
+
+import java.util.Collection;
+
+/**
+ * The type Collection utils.
+ */
+public class CollectionUtils {
+    /**
+     * Is not empty boolean.
+     *
+     * @param coll the coll
+     * @return the boolean
+     */
+    public static boolean isNotEmpty(Collection<?> coll) {
+        return null != coll && !coll.isEmpty();
+    }
+    
+    /**
+     * Is empty boolean.
+     *
+     * @param coll the coll
+     * @return the boolean
+     */
+    public static boolean isEmpty(Collection<?> coll) {

```

Comment:
better to rename this to `isNullOrEmpty` to explicitly declear its name.
And more, you can put this method into common module and combine it with `isNullOrEmpty` for `String` with generic.